tst/EnergyPlus/unit/WindowManager.unit.cc

```diff
@@ -136,6 +137,7 @@ TEST_F(EnergyPlusFixture, WindowFrameTest )
 
 	DataHeatBalance::ZoneIntGain.allocate(1);
 
+	createFacilityElectricPowerServiceObject();
```

Comment:
@EnergyArchmage Why should this be necessary for an envelope unit test? I noticed that this is used in several unit tests that don't relate to the Electric Power Service Manager. It seems that that shouldn't be necessary.